include/counter.h

```diff
@@ -29,6 +29,16 @@
 extern "C" {
 #endif
 
+/**@defgroup COUNTER_FLAGS Counter device capabilities
+ * @{ */
+
+/**
+ * @brief Counter count up flag.
+ */
+#define COUNTER_FLAG_COUNT_UP BIT(0)
```

Comment:
When using flags we need to ensure that it's clear to the user of the API which flags can be used where. And in the future there may be more functions / configuration structures using flags. The `COUNTER_FLAG_` prefix is rather ambiguous. I propose to use prefix `COUNTER_CONFIG_INFO_` as in `COUNTER_CONFIG_INFO_COUNT_UP`.

Reply:
done